testnets/epixtestnet/assetlist.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "epixtestnet",
+  "assets": [
+    {
+      "description": "The EPIX coin, the lifeblood of the Epix network, providing the utility that powers every interaction and transaction within the ecosystem.",
+      "denom_units": [
+        {
+          "denom": "atepix",
+          "exponent": 0
+        },
+        {
+          "denom": "tepix",
+          "exponent": 18
+        }
+      ],
+      "base": "atepix",
+      "name": "Epix Testnet",
+      "display": "tepix",
+      "symbol": "TEPIX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/epixtestnet/images/epix.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/epixtestnet/images/epix.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/epixtestnet/images/epix.png",
```

Comment:
Please remove the padding from the image files, as that won't be clipped in most UIs

Reply:
Uploaded new image.